### ISSUE TITLE
Prefer span over div for the default templte

### ISSUE THREAD

**Issue by oliviertassinari** [readthedocs/ethical-ad-client#18]
The following HTML structure is invalid: `p > div`, the following is valid: `p > span`.

**Comment by agjohnson** [comment]
I don't believe we output any `<p>` elements in our placements. Are you noticing otherwise, or is the case that you are surrounding the `<div>` in a `<p>` on your site?

We currently treat both text and image placements as a block elements anyways -- and both have nested block elements -- so a `<div>` is still probably the most semantically correct. If it's possible, it's probably best to just avoid the surrounding `<p>` element.

**Comment by oliviertassinari** [comment]
@agjohnson To provide more context on the problem, we render the as on a p element, it's a constraint we can't change. I have was leaning toward using the API raw, but it output a CORS issue.
